src/main/java/nextstep/ladder/domain/Participants.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Participants {
+    private static final String SPLIT_COMMA = ",";
+    private final List<Participant> participants = new ArrayList<>();
+
+    public Participants(String names) {
+        fromNames(names);
```

Comment:
```suggestion
        this(fromNames(names));
```
주 생성자를 호출하도록 구현할 것을 추천